module3/intermission_work/index.md

```diff
@@ -1,4 +1,81 @@
 ---
 layout: page
 title: Module 3 - Intermission Pre-Work
----
\ No newline at end of file
+---
+
+# Module 3 Intermission Work
+
+## Required Deliverables
+
+### You must submit all of these assignments. *They are due the Saturday before the start of the inning by 5pm*
+
+Submit your work here: [Survey Link](https://forms.gle/p2VnbLdA3QTVVC8N9)
+
+Versions:
+ - Ruby 3.2.2 (run `ruby  -v`)
+ - Rails 7.1.x (run `rails -v`)
+ - Node > v16 (run `node -v`, you are likely on the latest Node version by using `nvm`)
+
+If you need help updating Ruby or Rails versions, check out [this page](https://backend.turing.edu/module3/misc/ruby_and_rails_versions). If you need to revisit Node installation, check out [this page](https://mod0.turing.edu/computer-setup#install-node). 
+
+### 1. Technical Resume
+
+To start your Professional Development work in mod 3, plan to spend 60-90 minutes completing [this async lesson on crafting your Resume](https://docs.google.com/presentation/d/1HDtaMhRxM226YZWQqe93ILUpBIzDLzXFMEKOmjT1yUc/edit#slide=id.p1). 
+
+### 2. Consuming APIs with Ruby
+
+Complete [this exercise](../lessons/consuming_an_api_ruby.md) to gain familiarity with consuming an external API within your Set List application! 
```

Comment:
This link didn't work for me, but it did work if I removed the `.md` extension.

Reply:
My bad - fixing!